src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -153,7 +153,7 @@ public bool CanNavigateBackground
         /// <summary>
         /// A flag which indicates whether Isolate Selected Geometry mode is activated.
         /// </summary>
-        private bool isolationMode;
+        private bool isolationMode = false;
```

Comment:
This will be an enhancement that should be addressed in a subsequent pull request.